src/main/java/com/minecolonies/coremod/colony/crafting/ItemStorageFactory.java

```diff
@@ -52,7 +50,8 @@ public ItemStorage getNewInstance(@NotNull final ItemStack stack, final int size
     public CompoundNBT serialize(@NotNull final IFactoryController controller, @NotNull final ItemStorage storage)
     {
         final CompoundNBT compound = new CompoundNBT();
-        @NotNull final CompoundNBT stackTag = new CompoundNBT();
+        @NotNull CompoundNBT stackTag = new CompoundNBT();
+        Log.getLogger().info("Writing new ItemStorage: " + storage.getItemStack().getDisplayName().getString());
```

Comment:
debug logging, we should probably remove

Reply:
Oops. Missed that one